include/deal.II/cgal/triangulation.h

```diff
@@ -256,8 +271,8 @@ namespace CGALWrappers
             // A face is identified by a cell and by the index within the cell
             // of the opposite vertex. Loop over vertices, and retain only those
             // that belong to this face.
-            unsigned int j = 0;
-            for (unsigned int i = 0; i < 4; ++i)
+            int j = 0;
+            for (int i = 0; i < 4; ++i)
```

Comment:
What's the reason for changing the type here?

Reply:
I had a "comparison between signed and unsigned int" warning, as cgal indices in next lines are `int`s , so I removed `unsigned`.